ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -208,3 +290,10 @@ message DamlContractState {
 
 }
 
+// Stored information about a given party.
+message DamlPartyValue {
```

Comment:
I'd prefer a more descriptive name.. like DamlPartyAllocation? Not DamlPartyState as this is immutable.

Reply:
OK